src/layout/Sidebar/MenuLinks.tsx

```diff
@@ -157,7 +157,13 @@ export const MenuLinks: React.FC<Props> = ({ collapsed = false, onClick }) => {
       icon: Bridge,
       title: `Satellite ${t('header.bridge')}`,
       id: 'satellite-bridge'
-    }
+    },
+    {
+      link: 'https://www.hashport.network/',
+      icon: Bridge,
+      title: `Hashport ${t('header.bridge')}`,
+      id: 'hashport-bridge'
+    },
```

Comment:
please remove `,`

```suggestion
    }
```

its failing due to prettier formatting
![Screenshot 2023-02-06 at 3 19 59 PM](https://user-images.githubusercontent.com/6604146/216940009-9d9e77fc-1f76-4fa9-a108-b7569525617c.png)